docs/node/workflows.md

```diff
@@ -65,54 +65,81 @@ There is no official support for querying the state of asynchronously started Wo
 
 ## How to cancel a Workflow Execution
```

Comment:
This title doesn't reflect the content below.

To cancel a Workflow execution call the `cancel()` method on a `WorkflowStub`.

This section talks about cancellation of Workflow operations.

Reply:
```suggestion
## How Workflow operations are canceled
```